peru/main.py

```diff
@@ -221,6 +221,59 @@ def do_override(params):
             print('{}: {}'.format(module, params.runtime.get_override(module)))
 
 
+@peru_command('module', '''\
+Usage:
+    peru module add [<module>] [<url>] [<type>] [--rev=<rev>] [--reup=<reup>] 
+             [--filename=<filename>] [--sha=<sha>] [--unpack=<unpack>] [--yaml=<yaml>]
+    peru module --help
+
+Add a module to your Peru yaml file. Reguires a module name, url, and module type.
+
+Options:
+    add                       add new module
+      --rev=<rev>             set module rev value
+      --reup=<reup>           set module reup value
+      --filename=<filename>   set module filemame value
+      --sha=<sha>             set module sha1 value
+      --unpack=<unpack>       set module unpack value 
+      --yaml=<yaml>           file to use instead of peru.yaml     
```

Comment:
I think the `--file` and `--file-basename` flags that we already have at the top level might make `--yaml` unnecessary. (That said, currently specifying `--file` requires that you also specify `--sync-dir`. That might be overkill here. Not sure yet.)